### PR TITLE
BD-3043: Message-level one-click list-unsubscribe

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-3043**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

**Review by jaydubbyasee — APPROVED** [review]

File: _docs/_hidden/private_betas/one-click_list-unsubscribe.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+nav_title: Message-Level One-Click List-Unsubscribe Settings
+article_title: Message-Level One-Click List-Unsubscribe Settings
+permalink: "/one-click_unsubscribe/"
+hidden: true
+description: "This article provides an overview of the one-click list-unsubscribe header settings that are applied at a message level."
+---
+
+# Message-level one-click list-unsubscribe settings
+
+> Using one-click unsubscribe for the list-unsubscribe header ([RFC 8058](https://datatracker.ietf.org/doc/html/rfc8058)) focuses on providing an easy way for recipients to opt-out from emails. You can adjust this header setting to be applied at a message level in your emails.
```
Comment (rachel-feinberg) [inline _docs/_hidden/private_betas/one-click_list-unsubscribe.md:11]:
```suggestion
> The one-click unsubscribe for the list-unsubscribe header ([RFC 8058](https://datatracker.ietf.org/doc/html/rfc8058)) provides an easy way for recipients to opt-out from emails. You can adjust this header setting to be applied at a message level in your emails.
```

File: _docs/_hidden/private_betas/one-click_list-unsubscribe.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+{% alert important %}
+The message-level one-click list-unsubscribe setting is currently in early access. Contact your account manager if you're interested in participating in this early access.
+{% endalert %}
+
+In your email editor, go to **Sending Settings** > **Sending Info**. Select from the following options:
+
+* **Use workspace default**: Uses the **Email Unsubscribe Header** settings set in **Email Preferences**. Any changes made to this setting will apply to all messages.
+* **Unsubscribe globally from all emails**: Uses the Braze default one-click unsubscribe header. Users who click the unsubscribe button will have their global email subscription state set to "Unsubscribed."
```
Comment (rachel-feinberg) [inline _docs/_hidden/private_betas/one-click_list-unsubscribe.md:20]:
```suggestion
* **Unsubscribe globally from all emails**: Uses the Braze default one-click unsubscribe header. Users who click the unsubscribe button will have their global email subscription state set to "Unsubscribed".
```

File: _docs/_hidden/private_betas/one-click_list-unsubscribe.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+* **Unsubscribe from specific subscription group**: Uses the specified subscription group. Users who click the unsubscribe button will be unsubscribed from the selected subscription group
```
Comment (rachel-feinberg) [inline _docs/_hidden/private_betas/one-click_list-unsubscribe.md:21]:
```suggestion
* **Unsubscribe from specific subscription group**: Uses the specified subscription group. Users who click the unsubscribe button will be unsubscribed from the selected subscription group.
```

File: _docs/_hidden/private_betas/one-click_list-unsubscribe.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+* **Exclude unsubscribe**
+
+Adjusting this setting will override the default behavior for one-click list unsubscribe in this email.
+
+![]({% image_buster /assets/img/email_settings/one_click_list_unsubscribe_message_level.png %})
```
Comment (rachel-feinberg) [inline _docs/_hidden/private_betas/one-click_list-unsubscribe.md:26]:
```suggestion
![]({% image_buster /assets/img/email_settings/one_click_list_unsubscribe_message_level.png %}){: style="max-width:70%;"}
```

Or something smaller. The image seems kind of large.

File: _docs/_hidden/private_betas/one-click_list-unsubscribe.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+### Frequently asked questions
+
+#### If I add the email headers for one-click manually and I have email unsubscribe header turned on, what is the expected behavior?
+
+Select **Exclude unsubscribe** as the **One-click list-unsubscribe setting**. The email headers added for one-click list-unsubscribe will be applied to all future sends of this campaign.
```
Comment (rachel-feinberg) [inline _docs/_hidden/private_betas/one-click_list-unsubscribe.md:32]:
```suggestion
The email headers added for one-click list-unsubscribe will be applied to all future sends of this campaign.
```

Can we remove that first line? I'm unsure what it adds 🤔

File: _docs/_hidden/private_betas/one-click_list-unsubscribe.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+#### Is the one-click unsubscribe setting be available for email templates?
+
+No, we currently do not have plans to add this for email templates as these templates aren't assigned to a sending domain. If you're interested in this feature for email templates, submit [product feedback]({{site.baseurl}}/user_guide/administrative/access_braze/portal/).
+
+#### Does this feature check that the one-click unsubscribe URL added to the custom option is valid?
+
+No, we do not check or validate any links in the Braze dashboard.  
+
+No. This feature doesn't confirm if your custom one-click unsubscribe URL is working properly. Be sure to properly test your URL before launch.
```
Comment (rachel-feinberg) [inline _docs/_hidden/private_betas/one-click_list-unsubscribe.md:54]:
```suggestion
No, we don't check or validate any links in the Braze dashboard. Be sure to properly test your URL before launch.
```

I combined the above sentence with this one. It seems that one of the sentences was supposed to be deleted.

**Review by rachel-feinberg — APPROVED** [review]
Left a few comments!

File: _docs/_hidden/private_betas/one-click_list-unsubscribe.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+If a subscription group referenced in **Sending Info** for one-click is archived, Braze will still process unsubscribes from one-click. The subscription group will no longer be displayed on the dashboard (segment filter, user profile, and similar areas).
+
+#### Is the one-click unsubscribe setting be available for email templates?
+
+No, we currently do not have plans to add this for email templates as these templates aren't assigned to a sending domain. If you're interested in this feature for email templates, submit [product feedback]({{site.baseurl}}/user_guide/administrative/access_braze/portal/).
+
+#### Does this feature check that the one-click unsubscribe URL added to the custom option is valid?
+
+
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/one-click_list-unsubscribe.md:52]:
```suggestion
```

**Pull request merged**